README.md

```diff
@@ -552,11 +559,7 @@ method takes two arguments:
     1. if `f` is not a function, the behaviour of `reduce` is unspecified.
     2. The first argument to `f` must be the same type as `x`.
     3. `f` must return a value of the same type as `x`.
-    4. No parts of `f`'s return value should be checked.
-
-1. `x` is the initial accumulator value for the reduction
```

Comment:
why remove this line?

Reply:
I have no problem putting it back.  I just didn't see it adding anything.  Is there something that makes it clear that an "empty" foldable will return this value?  Do you think that this text captures that?